spec/controllers/assignment_invitations_controller_spec.rb

```diff
@@ -83,7 +77,7 @@
       it 'removes the repository on GitHub' do
         patch :accept_invitation, id: invitation.key
 
-        assert_requested :delete, github_url('/repositories/43894381')
+        assert_requested :delete, github_url('/repositories/44262216')
```

Comment:
I really wish we didn't have to do this. ref: https://github.com/education/classroom/pull/316#discussion_r41522602